docs/onboarding.md

```diff
@@ -26,6 +26,12 @@ Install dependencies
 npm install
 ```
 
+## 0. Register email with the Immutable Developer Hub
+
+Register your email address on the [Immutable Developer Hub](https://hub.immutable.com) to get access to customised documentation and the ability to create projects on Immutable.
```

Comment:
The wording feels like it can be interpreted as the hub offering BOTH customised docs and a UI to create projects